PatientRegistrationApp/src/components/HouseholdEmergencyContact.vue

```diff
@@ -63,43 +62,42 @@ export default {
     };
   },
   methods: {
-    /*sendHouseholdEmergencyContactInfoToHouseholdReviewPage()
+    sendHouseholdEmergencyContactInfoToReviewPage()
     {
-      const emergencyContactPayload = {
-        emergencyContactFamilyName: this.householdEmergencyContactFamilyName,
-        emergencyContactGivenName: this.householdEmergencyContactGivenName,
-        emergencyContactPhoneNumber: this.householdEmergencyContactPhoneNumber,
-        emergencyContactPhoneNumberType: this.householdEmergencyContactPhoneNumberType
+      const householdEmergencyContactPayload = {
+        householdEmergencyContactFamilyName: this.householdEmergencyContactFamilyName,
+        householdEmergencyContactGivenName: this.householdEmergencyContactGivenName,
+        householdEmergencyContactPhoneNumber: this.householdEmergencyContactPhoneNumber,
+        householdEmergencyContactPhoneNumberType: this.householdEmergencyContactPhoneNumberType
       }
-      EventBus.$emit('DATA_EMERGENCY_CONTACT_INFO_PUBLISHED', emergencyContactPayload)
-    },*/
+      EventBus.$emit('DATA_HOUSEHOLD_EMERGENCY_CONTACT_INFO_PUBLISHED', householdEmergencyContactPayload)
```

Comment:
Based on the name, this looks like a good example of application state!

Reply:
It's just a message to send household emergency contact data to the "review and submit" page.  If it's confusing, I can change PUBLISHED to ENTERED.  Until that message is able to be sent (required data has been entered), it is not possible to proceed in the workflow...so yes, it is technically a state, but I don't see a reason to add a separate application state variable to manage it.